dts/bindings/clock/nxp,imx-ccm.yaml

```diff
@@ -14,6 +14,21 @@ properties:
     "#clock-cells":
       const: 3
 
+    arm-div:
+      type: int
+      required: false
+      description: ARM clock divider
+
+    ahb-div:
+      type: int
+      required: true
+      description: AHB clock divider
+
+    ipg-div:
+      type: int
+      required: true
+      description: IPG clock divider
```

Comment:
When these values were encoded using Kconfigs, a range was enforced. Can you document the range here for future users?

Reply:
Sure, done.